dropwizard-version-info/src/test/java/com/palantir/dropwizard/versioninfo/VersionInfoBundleTests.java

```diff
@@ -76,13 +73,9 @@ public void testReturnsVersionFromDefaultPropertiesFile() {
         assertEquals(response, "2.0.0");
     }
 
-    @Test
+    @SuppressFBWarnings("NP_NULL_PARAM_DEREF_NONVIRTUAL")
+    @Test(expected = IllegalArgumentException.class)
     public void testInvalidPath() {
```

Comment:
this is for simple catching where as the other one has logic in the catch block. Personally i prefer this style if it is a simple catch block and having to call fail method explicitly is more error-prone